src/main/java/org/opensearch/flowframework/rest/RestDeleteWorkflowAction.java

```diff
@@ -92,7 +93,9 @@ protected BaseRestHandler.RestChannelConsumer prepareRequest(RestRequest request
                     XContentBuilder exceptionBuilder = ex.toXContent(channel.newErrorBuilder(), ToXContent.EMPTY_PARAMS);
                     channel.sendResponse(new BytesRestResponse(ex.getRestStatus(), exceptionBuilder));
                 } catch (IOException e) {
-                    String errorMessage = "IOException: Failed to send back delete workflow exception";
+                    String errorMessage = ParameterizedMessageFactory.INSTANCE.newMessage(
+                        "IOException: Failed to send back delete workflow exception"
```

Comment:
see above; no need to replace a plain string but consider parameterizing "delete"